common/util/file_util.cc

```diff
@@ -46,6 +48,39 @@ absl::string_view Stem(absl::string_view filename) {
              : filename.substr(0, last_dot_pos);
 }
 
+bool UpwardFileSearch(absl::string_view start, absl::string_view filename,
+                      std::string& result) {
+  /* Convert to absolute path */
+  char absolute_path[PATH_MAX];
+  if (realpath(std::string(start).c_str(), absolute_path) < 0) {
+    return false;
+  }
+
+  /* Add trailing slash */
+  std::string init = absl::StrCat(absolute_path, "/");
```

Comment:
Could we use `std::filesystem::path::preferred_separator` for portability?  (ok to make a short local alias for that)

Reply:
Sure, however the README file says:
> To build, you need the bazel build system and a C++11 compatible compiler (...)

To use your suggestion we would need `c++17`.

Reply:
The next best thing may be to define this locally as a constant.

Reply:
I think that the whole function could be rewritten to use `std::filesystem` - I added a comment with TODO